sdks/python/apache_beam/transforms/enrichment_handlers/feast_feature_store.py

```diff
@@ -0,0 +1,167 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import logging
+import tempfile
+from pathlib import Path
+from typing import List
+from typing import Optional
+
+import apache_beam as beam
+from apache_beam.io.gcp.gcsfilesystem import GCSFileSystem
+from apache_beam.transforms.enrichment import EnrichmentSourceHandler
+from apache_beam.transforms.enrichment_handlers.utils import ExceptionLevel
+from feast import FeatureStore
+
+__all__ = [
+    'FeastFeatureStoreEnrichmentHandler',
+]
+
+_LOGGER = logging.getLogger(__name__)
+
+LOCAL_FEATURE_STORE_YAML_FILENAME = 'fs_yaml_file.yaml'
+
+
+def download_fs_yaml_file(gcs_fs_yaml_file: str):
+  """Download the feature store config file for Feast."""
+  try:
+    fs = GCSFileSystem(pipeline_options={})
```

Comment:
Rather than using GCSFileSystem, could we just use Beam's FileSystem module so that its location independent? Something like https://github.com/apache/beam/blob/8454cc9503008e28f5d3e90d39659352d46af9c2/sdks/python/apache_beam/ml/inference/pytorch_inference.py#L99